src/net/cert/internal/system_trust_store.cc

```diff
@@ -325,23 +325,27 @@ std::unique_ptr<SystemTrustStore> CreateSslSystemTrustStore() {
 
 namespace {
 
-// Copied from https://golang.org/src/crypto/x509/root_linux.go
+// Copied from https://go.dev/src/crypto/x509/root_linux.go
 // Possible certificate files; stop after finding one.
-constexpr std::array<const char*, 6> kStaticRootCertFiles = {
-    "/etc/ssl/certs/ca-certificates.crt",  // Debian/Ubuntu/Gentoo etc.
-    "/etc/pki/tls/certs/ca-bundle.crt",    // Fedora/RHEL 6
-    "/etc/ssl/ca-bundle.pem",              // OpenSUSE
-    "/etc/pki/tls/cacert.pem",             // OpenELEC
+constexpr const char* kStaticRootCertFiles[] = {
+    "/etc/ssl/certs/ca-certificates.crt",                 // Debian/Ubuntu/Gentoo etc.
+    "/etc/pki/tls/certs/ca-bundle.crt",                   // Fedora/RHEL
+    "/etc/ssl/ca-bundle.pem",                             // OpenSUSE
+    "/etc/pki/tls/cacert.pem",                            // OpenELEC
     "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem",  // CentOS/RHEL 7
     "/etc/ssl/cert.pem",                                  // Alpine Linux
 };
 
 // Possible directories with certificate files; stop after successfully
 // reading at least one file from a directory.
-constexpr std::array<const char*, 3> kStaticRootCertDirs = {
-    "/etc/ssl/certs",      // SLES10/SLES11, https://golang.org/issue/12139
-    "/etc/pki/tls/certs",  // Fedora/RHEL
-    "/system/etc/security/cacerts",  // Android
+constexpr const char* kStaticRootCertDirs[] = {
+    "/etc/ssl/certs",                                  // SLES10/SLES11, https://golang.org/issue/12139
+    "/etc/pki/tls/certs",                              // Fedora/RHEL
+    "/etc/pki/ca-trust/extracted/pem/directory-hash",  // Fedora/RHEL 9.5/10
+#if BUILDFLAG(IS_ANDROID)
+    "/system/etc/security/cacerts",                    // Android system roots
+    "/data/misc/keychain/certs-added",                 // User trusted CA folder
```

Comment:
Not seeing justification for this addition

I prefer not allowing user added certs.

Reply:
![image](https://github.com/user-attachments/assets/7d8be751-43f7-492f-91a5-f06987440132)

It is your choice but it is recommended by golang project.

Reply:
This one  can be left alone

Reply:
See comments above.